pwndbg/commands/context.py

```diff
@@ -833,6 +833,67 @@ def context_args(with_banner=True, target=sys.stdout, width=None):
 
 last_signal: List[str] = []
 
+thread_status_messages = {
+    "running": pwndbg.color.light_green("running"),
+    "stopped": pwndbg.color.yellow("stopped"),
+    "exited": pwndbg.color.gray("exited "),
```

Comment:
the space is probably not needed
```suggestion
    "exited": pwndbg.color.gray("exited"),
```

Reply:
This space is needed for padding because `stopped` has one more character than `exited`